charts/m4d/templates/opa_connector.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: opa-connector
+  labels:
+    app: opa-connector
+spec:
+  selector:
+    matchLabels:
+      app: opa-connector
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: opa-connector
+        m4d.ibm.com/componentType: connector
+    spec:
+      containers:
+      - name: opa-connector
+        image: {{ .Values.image.opaConnector }}
+        imagePullPolicy: Always
```

Comment:
Can we also have a variable for this? Especially for production environments `IfNotPresent` might be a better option. That way it's configurable.

Reply:
sure